README.md

```diff
@@ -28,7 +28,7 @@ Create different kinds of build scans by locally modifying this quickstart proje
 
 - Edit `src/main/scala/example/Example.scala` to introduce compile errors
 
-Alternatively, enable one of your own sbt builds to produce build scans by following the [step-by-step instructions][scans.gradle.com].
+Alternatively, enable one of your own sbt builds to produce build scans by following the [step-by-step instructions][scans.gradle.com/#sbt].
```

Comment:
I tried to click the link in the branch and the anchor doesn't seem to work.

Reply:
ah, thanks. Let me fix this!